### DISCUSSION TITLE
Welcome to CTP_01_TableReader Discussions!

### DISCUSSION THREAD

**Discussion by HolgerSin** [TeamOfStudents/CTP_01_TableReader#4] [Announcements]
## 👋 Willkommen
  
Ob wir die "Discussions" Funktionen im Rahmen dieses Übungsprojekts aktiv nutzen werden wird sich im Lauf des Projekts zeigen. 

Für den Anfang ist das Modul zunächst mal freigeschaltet, und Ihr könnt hier gerne alle projektbezogenen Themen posten die Euch in den Sinn kommen. 

Wenn Ihr grundsätzlich Interesse habt Euch im Rahmen dieses **CTP_01_TableReader** Repositories zu beteiligen, lasst mir doch einen thumbs up (oder einen Kommentar) da, damit ich weiß wer Interesse hat.

Edit:
Mache hier erstmal provisorisch eine Liste derjenigen, die aktiv mitarbeiten wollen:
@mauricekoenig 
@RabuMia 
@nicopape 
@danjan09 
@Hostmisha1 
@SnickErzIsBack 
@fatihkoeksal 
@Indu1284 

updated 28.08.

Wenn sich sonst noch jemand einbringen will, bitte hier einen Kommentar hinterlassen.

**Comment by mauricekoenig** [comment]
@SnickErzIsBack
